src/components/UrqlMutation.re

```diff
@@ -0,0 +1,55 @@
+[@bs.module "urql"] external mutation: ReasonReact.reactClass = "Mutation";
+let component = ReasonReact.statelessComponent("Mutation");
+
+type mutationRenderPropsJs('a) = {
+  .
+  "fetching": bool,
+  "data": Js.Nullable.t('a),
+  "error": Js.Nullable.t(UrqlError.t),
+  "executeMutation": Js.Json.t => unit,
```

Comment:
In reviewing the `urql` docs, I think this actually returns a `Promise`. cc/ @kitten

Reply:
That's correct 👌